tutorials/napps/switch_l3_part1.rst

```diff
@@ -0,0 +1,426 @@
+Running and testing your NApp
+*****************************
+
+To run your NApp, you need to run *Kytos* first to enable NApp management. In
+another terminal window, make sure to activate your |dev_env|_ and run:
+
+.. code-block:: bash
+
+  $ kytosd -f
+  2017-07-25 14:45:35,135 - INFO [kytos.core.logs] (MainThread) Logging config file "/home/user/test42/etc/kytos/logging.ini" loaded successfully.
+  2017-07-25 14:45:35,137 - INFO [kytos.core.controller] (MainThread) /home/user/test42/var/run/kytos
+  2017-07-25 14:45:35,137 - INFO [kytos.core.controller] (MainThread) Starting Kytos - Kytos Controller
+  2017-07-25 14:45:35,139 - INFO [kytos.core.tcp_server] (TCP server) Kytos listening at 0.0.0.0:6633
+  2017-07-25 14:45:35,142 - INFO [kytos.core.controller] (RawEvent Handler) Raw Event Handler started
+  2017-07-25 14:45:35,144 - INFO [kytos.core.controller] (MsgInEvent Handler) Message In Event Handler started
+  2017-07-25 14:45:35,148 - INFO [kytos.core.controller] (MsgOutEvent Handler) Message Out Event Handler started
+  2017-07-25 14:45:35,148 - INFO [kytos.core.controller] (AppEvent Handler) App Event Handler started
+  2017-07-25 14:45:35,148 - INFO [kytos.core.controller] (MainThread) Loading Kytos NApps...
+  2017-07-25 14:45:35,153 - INFO [kytos.core.napps.napp_dir_listener] (MainThread) NAppDirListener Started...
+  2017-07-25 14:45:35,155 - INFO [kytos.core.controller] (MainThread) Loading NApp kytos/of_core
+  2017-07-25 14:45:35,612 - INFO [root] (kytos/of_core) Running NApp: <Main(kytos/of_core, started 140029615662848)>
+
+  (...)
+
+  kytos $>
+
+As you can see, there is a log line indicating that *kytos/of_core* is running.
+You need the OpenFlow core NApp installed and enabled. It is possible to check
+it by running, in the previous terminal window:
+
+.. code-block:: bash
+
+  $ kytos napps list
+
+  Status |          NApp ID          |                     Description
+  =======+===========================+======================================================
+   [ie]  | kytos/of_core             | OpenFlow Core of Kytos Controller, responsible for...
+   [i-]  | kytos/of_flow_manager     | Manage switches' flows through a REST API.
+   [i-]  | kytos/of_ipv6drop         | Install flows to DROP IPv6 packets on all switches.
+   [i-]  | kytos/of_l2ls             | An L2 learning switch application for OpenFlow swi...
+   [i-]  | kytos/of_lldp             | Discovers switches and hosts in the network using ...
+   [i-]  | kytos/of_stats            | Provide statistics of openflow switches.
+   [i-]  | kytos/of_topology         | Keeps track of links between hosts and switches. R...
+   [i-]  | kytos/web_topology_layout | Manage endpoints related to the web interface sett...
+
+If the NApp is installed but not enabled, you can enable it by running:
+
+.. code-block:: bash
+
+  $ kytos napps enable kytos/of_core
+
+Now, install and run the *l3ls* NApp:
+
+.. code-block:: bash
+
+  $ kytos napps install tutorial/of_l3ls
```

Comment:
@renanrodrigo at this point, the user may be under the `tutorial/of_l3ls` directory. `kytos napps install` will be able to install the napp from within the napp directory?